### PR TITLE
Profit aware grid - Ruby Curve

### PR DESCRIPTION
**Pull request by Siddharth2207**
## Motivation
Ruby strategy buy and sell curve.

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [ ] unit-tested any new functionality
- [ ] linked any relevant issues or PRs

### CODE REVIEW

File: strategies/ruby/buy-curve.rain [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+deployments:
+  h20-ruby-buy:
+    scenario: buy-curve.prod
+    order: base-ruby-buy     
+---
+#orderbook-subparser !The address for uniswap subparser.
+#uniswap-words !The address for orderbook subparser. 
+#scale-factor !Scale factor for buy curve.
+
+#in-token-key 0x10b2c7dd2f0c383e8445066430f62720e189027b16a56ad4212eefac8e6c77ed
```
Comment (thedavidmeister) [inline strategies/ruby/buy-curve.rain]: what's going on with all these hardcoded chunks of bytes?

File: strategies/ruby/buy-curve.rain [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+#plottables-prod
+  :;
+
+#plottables-chart
+  _: scale-factor;
+
+#calculate-io
+ using-words-from orderbook-subparser uniswap-words
+
+ out-tkn-amount: get(0x469017709316f7f258096e4e2ac47b89e1a104ffa810dcd5c906f95726cdbf78),
```
Comment (thedavidmeister) [inline strategies/ruby/buy-curve.rain]: why isn't this using the key binding that looks like it's supposed to be for this?

File: strategies/ruby/sell-curve.rain [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+#end-mc !Market capitalization for the token at the end of the token sale.
+
+#initial-token-amount !Initial amout offered for sale.
+#sale-token-address !Address of sale token.
+
+#plottables !Bindings for the plottable source.
+#ensure-sale-start !Binding to ensure sale has began.
+#test-now !Binding to simulate current time.
+
+#in-token-key 0x10b2c7dd2f0c383e8445066430f62720e189027b16a56ad4212eefac8e6c77ed
```
Comment (thedavidmeister) [inline strategies/ruby/sell-curve.rain]: same as above

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: strategies/ruby/buy-curve.rain (hunk 2 above)
Comment (Siddharth2207) [reply strategies/ruby/buy-curve.rain]: Resolved

File: strategies/ruby/sell-curve.rain (hunk 3 above)
Comment (Siddharth2207) [reply strategies/ruby/sell-curve.rain]: Resolved

File: strategies/ruby/buy-curve.rain (hunk 1 above)
Comment (Siddharth2207) [reply strategies/ruby/buy-curve.rain]: Resolved

**Comment by thedavidmeister** [comment]
this is no longer relevant for any active customers, if we need it we can dust it off for a specific use case when we have specific requirements

**Pull request closed without merging**